ros_docker_images/templates/docker_images/create_base_image.Dockerfile.em

```diff
@@ -26,10 +19,16 @@ ENV TZ @timezone
 
 # install bootstrap tools
 RUN apt-get update && apt-get install -q -y \
+    locales \
     python-rosdep \
     python-rosinstall \
     python-vcstools
 
+# setup environment
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV TZ @timezone
```

Comment:
Oh, this 'base' template isn't used in the official images is it. Is it for the build farm? That fine then right?